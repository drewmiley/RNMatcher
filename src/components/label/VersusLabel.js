import React, { Component } from 'react';
import { Text } from 'react-native';

export default class VersusLabel extends Component {
    constructor(props) {
        super(props);
        this.shouldComponentUpdate = false;
    };
    render() {
        return <Text> Versus </Text>
    };
};