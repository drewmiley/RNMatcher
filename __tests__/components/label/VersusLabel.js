import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import VersusLabel from '../../../src/components/label/VersusLabel';

describe('VersusLabel', () => {

    describe('Rendering', () => {

        it('should match the snapshot', () => {
            const component = renderer.create(
                <VersusLabel />
            );            
            expect(component).toMatchSnapshot();
        });

    });

});