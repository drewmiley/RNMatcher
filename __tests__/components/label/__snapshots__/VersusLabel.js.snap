exports[`VersusLabel Rendering should match the snapshot 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail">
   Versus 
</Text>
`;
